tests/commands/envelope_test.py

```diff
@@ -355,3 +356,41 @@ def test_apply_from_user_and_email(self):
 
         self.assertTrue(env.sign)
         self.assertIs(env.sign_key, mock.sentinel.gpg_key)
+
+
+class TestSendCommand(unittest.TestCase):
```

Comment:
Agreed on dropping twisted. Note also that those classes use `twisted.trial.unittest` not `unittest`